code/game/jobs/job/security.dm

```diff
@@ -137,6 +137,9 @@
 
 /datum/outfit/job/detective/pre_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
 	. = ..()
+	if(visualsOnly)
```

Comment:
Nope, this actually shouldn't be here- all that's below is changing the detective's outwards appearance.

Reply:
runtime says otherwise [13:22:06] Runtime in security.dm,143: Cannot read null.role_alt_title